recipes/cutadapt/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   md5: 222d062207e2a3a2a0760caa83bf14ff
 
 build:
-  number: 0
-  script: $PYTHON setup.py install
+  number: 1
+  script: python -m pip install --no-deps .
```

Comment:
Good change!
`pip install --no-deps .`
would also work. Can you add `pip` to the build requirements, just for completeness' sake?